doc/lsp.txt

```diff
@@ -333,7 +333,9 @@ accepts a list of LSP servers with the above information.
 
 						*:LspHover*
 :LspHover		Show the documentation for the symbol under the cursor
-			in a popup window.
+			in a popup window. If you prefer show documentation in
+			a preview window rather a popup set
```

Comment:
Can you change the text to "If you want to show the symbol documentation in the preview window instead of in a popup, set the g:LSP_Hover_In_preview variable to v:true?

Reply:
Done.